proposals/64:crabble-start/assets/crabbleCoreEval.js

```diff
@@ -0,0 +1,250 @@
+      namesByAddressAdmin,
+      startUpgradable,
+    },
+    installation: {
+      consume: { committee: committeeInstallationP },
+    },
+    instance: { produce: produceInstance },
+  },
+  config,
+  crabbleNode,
+) => {
+  const committeeSize = 3;
+  const committeeName = "CrabbleCommittee";
+  const members = ["agoric1ag5a8lhn00h4u9h2shpfpjpaq6v4kku54zk69m","agoric140y0mqnq7ng5vvxxwpfe67988e5vqar9whg309","agoric1wqfu6hu5q2qtey9jtjapaae4df9zd492z4448k"];
+
+  logger('Getting nameHubs, depositFacets...');
+  const getDepositFacet = async (address) => {
+    const hub = E(E(namesByAddressAdmin).lookupAdmin(address)).readonly();
+    return E(hub).lookup('depositFacet');
+  };
+  const memberDeposits = await Promise.all(members.map(getDepositFacet));
+
+  logger('Gathering info...');
+  const { committeeInstallation, marshaller, committeeNode } = await allValues({
+    committeeInstallation: committeeInstallationP,
+    marshaller: E(board).getPublishingMarshaller(),
+    committeeNode: E(crabbleNode).makeChildNode('committee'),
+  });
+
+  logger('Starting committee...');
+  const committeeKit = await E(startUpgradable)({
+    installation: committeeInstallation,
+    terms: { committeeSize, committeeName },
+    privateArgs: { storageNode: committeeNode, marshaller },
+    label: committeeName,
+  });
+  logger({ committeeKit });
+
+  logger('Updating agoricNames with committee instance...');
+  produceInstance.CrabbleCommittee.resolve(committeeKit.instance);
+
+  logger('Getting the member and voter invitations...');
+  const voterInvitations = await E(
+    committeeKit.creatorFacet,
+  ).getVoterInvitations();
+
+  logger('Sending committeeinvitations...');
+  await Promise.all(
+    zip(memberDeposits, voterInvitations).map(([depositFacet, invitation]) =>
+      E(depositFacet).receive(invitation),
+    ),
+  );
+
+  logger('Done.');
+  return { committeeCreatorFacet: committeeKit.creatorFacet, memberDeposits };
+};
+
+/**
+ *
+ * @param {{
+ *   consume: {
+ *     zoe: Promise<ZoeService>;
+ *     board: ERef<XMPORT('@agoric/vats').Board>,
+ *     startCrabbleGovernedUpgradable: Promise<Function>,
+ *     chainTimerService: ERef<XMPORT('@agoric/time/src/types').TimerService>;
+ *     agoricNames: ERef<XMPORT('@agoric/vats').NameHub>;
+ *   },
+ *   instance: { produce: Record<'Crabble' | 'CrabbleGovernor', ProducerX<Instance>>}
+ * }} powers
+ * @param {*} config
+ * @param {ERef<StorageNode>} crabbleNode
+ * @param {Promise<{
+ *   committeeCreatorFacet: ERef<any>;
+ *   memberDeposits: ERef<DepositFacet>[]
+ * }>} committeeInfoP
+ */
+const startCrabble = async (powers, config, crabbleNode, committeeInfoP) => {
+  const contractBundleID = "b1-145f84363c784c96bc5a5d456edf1853bf2bb04388a250f010a53234b333af46d8d08e3c58d6e3bac61c45b18e82a09199938466616016a22d9191448f8e505c";
+  const governorBundleID = "b1-d67b9ac5100bf45a3369d393fe2cb38e0718f770cac936292a18270ed331865ffa817719f5c6ceacdb1a580572d71084db0545e8e13efaee38c7df76f51f756c";
```

Comment:
As noted in https://github.com/Agoric/agoric-3-proposals/pull/41#issuecomment-1852492922 , I'm not able to reproduce these bundle hashes.

Reply:
Here's how everything happened when I tried it:

```bash
anil@Anils-MacBook-Pro source_code % make prepare-evals
node /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/contract/src/proposals/prepareCoreEval.js
{
  OUT_DIR_ROOT: '/Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code',
  RENDER_EVALS: undefined
}
{
  value: [
    'agoric1ag5a8lhn00h4u9h2shpfpjpaq6v4kku54zk69m',
    'agoric140y0mqnq7ng5vvxxwpfe67988e5vqar9whg309',
    'agoric1wqfu6hu5q2qtey9jtjapaae4df9zd492z4448k'
  ]
}
{ value: 3 }
{ value: 'CrabbleCommittee' }
[bundleTool] /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/bundles add: crabble from /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/contract/src/crabble.js 
[bundleTool] /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/bundles bundled 205 files in bundle-crabble.js at 2023-12-12T20:07:51.192Z 
[bundleTool] /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/bundles add: crabbleGovernor from /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/contract/src/crabbleGovernor.js 
[bundleTool] /Users/anil/WebstormProjects/agoric-samples/a3p-bundle-experiment/crabbleProtocol/source_code/bundles bundled 137 files in bundle-crabbleGovernor.js at 2023-12-12T20:07:52.208Z 
DONE
anil@Anils-MacBook-Pro source_code % diff -ur assets ../../../crabbleProtocol/source_code/assets
anil@Anils-MacBook-Pro source_code % diff -ur assets ../../../crabbleProtocol/source_code/assets | cut -c 1-200
anil@Anils-MacBook-Pro source_code % jq ".endoZipBase64Sha512" assets/bundle-contract.json
"145f84363c784c96bc5a5d456edf1853bf2bb04388a250f010a53234b333af46d8d08e3c58d6e3bac61c45b18e82a09199938466616016a22d9191448f8e505c"
anil@Anils-MacBook-Pro source_code % jq ".endoZipBase64Sha512" assets/bundle-governor.json 
"d67b9ac5100bf45a3369d393fe2cb38e0718f770cac936292a18270ed331865ffa817719f5c6ceacdb1a580572d71084db0545e8e13efaee38c7df76f51f756c"
```

Is there any chance you forgot to check out the commit hash I provided in the release docs? @dckc 

```sh
git checkout 0cb86ed3ce19f211ee033fcad9fa95ce2097e0d3
```